src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -1501,14 +1529,24 @@ METHOD(kernel_ipsec_t, add_terminator_conn, status_t,
 		 .this = this,
 		 .term_unique_id = picomsg_terminator_conn_get_unique_id(new_term_conn),
 	);
-
 	lib->scheduler->schedule_job_ms(lib->scheduler,
 									(job_t*)callback_job_create_with_prio(terminator_ka_timer_cb,
 																		  ka_timer_ctx, free, NULL, JOB_PRIO_HIGH),
 									TERMINATORS_KA_TIMER_MS);
 
 	picomsg_terminator_conn_unref(new_term_conn);
 
+	struct terminator_count_tunnels_timer_cb_ctx* tunnel_counters_ctx;
+	INIT(tunnel_counters_ctx,
+		 .this = this,
+		 .term_unique_id = picomsg_terminator_conn_get_unique_id(new_term_conn),
+	);
+
+	lib->scheduler->schedule_job_ms(lib->scheduler,
```

Comment:
The requirement is to write to oper every 10 seconds, while this will write every 10 seconds the oper information of each terminator since its creation. This means that depending on the timings of terminators creation we might write every say 5 seconds the oper info of just one terminator.

Instead need to have a single timer callback that will iterate over the terminators write their oper info